skipper/cli.py

```diff
@@ -154,10 +154,11 @@ def rmi(ctx, remote, image, tag):
 
 @cli.command(context_settings=dict(ignore_unknown_options=True))
 @click.option('-i', '--interactive', help='Interactive mode', is_flag=True, default=False, envvar='SKIPPER_INTERACTIVE')
+@click.option('-n', '--name', help='container name', default=None)
```

Comment:
dunno.
From this point you are right but logically this place is better as for me.
@eran-stratoscale what do u think?